### PR TITLE
Docs 9858 x12 connector 2.5 example rework

### PR DESCRIPTION
**Pull request by mcherem**

### CODE REVIEW

File: x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc [hunk 1]
```diff
@@ -1,136 +1,43 @@
...
 
-== Read and Validate X12 EDI Messages
+* Transforms XML messages into Outbound EDI X12 850 purchase order transactions +
++
+* Transforms Inbound EDI X12 850 purchase orders into JSON +
++
+* Generates a Functional acknowledgment (997) +
 
-. To read an X12 message, search the palette for `X12 EDI:Configuration` and drag the X12 Read operation into a flow.
+An HTTPS Listener is used as the message source to receive the Enterprise purchase order XML messages from a back-end system. DataWeave and X12 Writes are used to translate the XML message into an X12 v4010 850 Purchase order EDI message. A (997) Functional Acknowledgment is generated upon receipt of the purchase order message.
```
Comment (dhirsch1) [inline x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc]: Suggestion to remove some of the passive voice:

An HTTPS Listener receives the...

DataWeave and X12 Writes translate the...

**Review by dhirsch1 — DISMISSED** [review]

File: x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc (hunk 1 above)
Comment (mcherem) [reply x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc]: Fixed, thanks.

File: x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc [hunk 2]
```diff
@@ -1,138 +1,51 @@
 = X12 EDI Connector 2.5 - Examples - Mule 4
 
-The following examples show how to read, validate, and write EDI messages. In addition, you can send functional and interchange acknowlegments to your trading partners.
+Use this example to view the use case for Business-to-Business (B2B) EDI integration leveraging Anypoint X12 Connector. This example:
 
-== Read and Validate X12 EDI Messages
+* Transforms XML messages into Outbound EDI X12 850 purchase order transactions +
 
-. To read an X12 message, search the palette for `X12 EDI:Configuration` and drag the X12 Read operation into a flow.
+An HTTPS Listener message source receives the Enterprise purchase order XML messages from a back-end system. DataWeave and X12 Writes translate the XML message into an X12 v4010 850 Purchase order EDI message.
```
Comment (dhirsch1) [inline x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc]: An HTTPS Listener message source > An HTTPS Listener source

File: x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc [hunk 3]
```diff
@@ -1,138 +1,51 @@
...
-        }
-    }
-}]]></ee:set-payload>
-			</ee:message>
-		</ee:transform>
-		<x12:write doc:name="Write" config-ref="X12_EDI_Config"/>
-	</flow>
-</mule>
-----
+An HTTP Listener message source receives the raw EDI X12 850 purchase order message.
```
Comment (dhirsch1) [inline x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-examples.adoc]: Same comment

**Review by dhirsch1 — DISMISSED** [review]

**Review by emileemooreVT — APPROVED** [review]

**Pull request merged**